src/components/common/icon/index.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { FC, ReactNode } from "react";
-import Apple from "./Apple";
-import Facebook from "./Facebook";
-import Logo from "./Logo";
-import Menu from "./Menu";
-import Twitter from "./Twitter";
+import Apple from "@assets/icons/Apple";
+import Facebook from "@assets/icons/Facebook";
+import Logo from "@assets/icons/Logo";
+import Menu from "@assets/icons/Menu";
+import Twitter from "@assets/icons/Twitter";
+import Close from "@assets/icons/Close";
```

Comment:
Perhaps you could export all the icons together and then import them like 
`import {Apple, Facebook} from "@assets/icons"`

Reply:
Way cleaner, thanks!